gen-bundle-image.sh

```diff
@@ -67,6 +74,7 @@ function sign_bundle_files {
   rm -fr *.sig
   if [[ ${preset} != "okd" ]]; then
      gpg --batch --default-key crc@crc.dev --pinentry-mode=loopback --passphrase-file ${GPG_SECRET_KEY_PASSPHRASE_PATH} --armor --output ${vfkit_bundle_arm64}.sig --detach-sig ${vfkit_bundle_arm64}
+     gpg --batch --default-key crc@crc.dev --pinentry-mode=loopback --passphrase-file ${GPG_SECRET_KEY_PASSPHRASE_PATH} --armor --output ${libvirt_bundle_arm64}.sig --detach-sig ${libvirt_bundle_arm64}
```

Comment:
should this be outside of the `okd` check, since image will be generated also for okd?

Reply:
No, we are not having arm64 support for okd.